tests/test_generate_success.py

```diff
@@ -0,0 +1,61 @@
+# Global imports
+"""Test the generate_config function in a perfect situation."""
+import pytest
+from gatorgrade.generate.generate import generate_config
+
+
+@pytest.fixture(name="testing_dir")
+def setup_files(tmp_path):
+    """Setup directory for tests"""
+    # Given this file structure
+    src_dir = tmp_path / "src"
+    src_dir.mkdir()
+
+    python_file = src_dir / "test.py"
+    python_file.write_text("import sys")
+
+    writing_dir = tmp_path / "writing"
+    writing_dir.mkdir()
+
+    reflection = writing_dir / "reflection.md"
+    reflection.write_text("# Reflection")
+    yield tmp_path
+
+
+def test_generate_config_create_gatorgrade_yml(testing_dir):
+    """Test to see that gatorgrade_yml file exist in file structure"""
+    # When generate_config is called
+    generate_config(["src"], testing_dir)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = testing_dir / "gatorgrade.yml"
+    assert gatorgrade_yml.is_file()
+
+
+def test_generate_config_creates_gatorgrade_yml_with_dir_in_user_input(testing_dir):
+    """Test to see if input matches directory"""
+    # When generate_config is called
+    generate_config(["src"], testing_dir)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = testing_dir / "gatorgrade.yml"
+    assert "src/test.py" in gatorgrade_yml.open().read()
+
+
+def test_generate_config_creates_gatorgrade_yml_without_dir_not_in_user_input(
+    testing_dir,
+):
+    """Test to see if input does not match directory"""
+    # When generate_config is called
+    generate_config(["writing"], testing_dir)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = testing_dir / "gatorgrade.yml"
+    assert "src/test.py" not in gatorgrade_yml.open().read()
+    assert "writing/reflection.md" in gatorgrade_yml.open().read()
+
+
+def test_generate_success_message(capsys, testing_dir):
+    """Test to see that there is a success message"""
+    # When generate_config is called
+    generate_config(["src"], testing_dir)
+    out = capsys.readouterr()
+    # Then a success message is printed
+    assert "SUCCESS" in out
```

Comment:
`capsys.readouterr()` returns a named tuple that looks like `(out, err)` according to its [documentation](https://docs.pytest.org/en/6.2.x/capture.html#accessing-captured-output-from-a-test-function)-- so either you should deconstruct it here (replace `out` with `out, err`, or use a different name (like `captured`) and then access the standard output with `captured.out` for assertions.

Reply:
Oh, I see. Thanks! I went ahead and talk to Professor Maria and got some things corrected. Although I still seem to get the linting issue for `windows-latest`. Do you happen to know how to fix the issue?